### PR TITLE
.NET SDK for Oracle.Database

### PR DESCRIPTION
**Pull request by v-jamcheung**
.NET SDK for Oracle.Database

Library Namespace: Azure.ResourceManager.OracleDatabase (https://github.com/Azure/azure-sdk/issues/7507)
Merged PR for Oracle.Database Stable Spec: https://github.com/Azure/azure-rest-api-specs/pull/28090

### CODE REVIEW

**Review by jsquire — CHANGES_REQUESTED** [review]
@v-jamcheung:

Your account lacks the required public GitHub organizations and permissions required of an internal contributor.  Please review the Azure SDK onboarding documentation and use the associated Teams channel for support.
- [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
- [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_

You can verify the state of your account by running the [Validate-AzsdkCodeOwner](https://github.com/Azure/azure-sdk-tools/blob/main/tools/github/scripts/Validate-AzsdkCodeOwner.ps1) script from the Azure SDK tools repository.

Please also be sure to add yourself to [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS) for this library, if you will be maintaining it going forward.

File: .github/CODEOWNERS
Comment (v-jamcheung) [reply .github/CODEOWNERS]: Updated

**Comment by v-jamcheung** [comment]
> @v-jamcheung:
> 
> Your account lacks the required public GitHub organizations and permissions required of an internal contributor. Please review the Azure SDK onboarding documentation and use the associated Teams channel for support.
> 
> * [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
> * [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_
> 
> You can verify the state of your account by running the [Validate-AzsdkCodeOwner](https://github.com/Azure/azure-sdk-tools/blob/main/tools/github/scripts/Validate-AzsdkCodeOwner.ps1) script from the Azure SDK tools repository.
> 
> Please also be sure to add yourself to [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS) for this library, if you will be maintaining it going forward.

Updated [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS).

As for the permissions, I believe I will not need write permissions as someone else will be assigned this PR and will merge it once I have addressed all the comments and gotten the necessary approvals.

**Comment by jsquire** [comment]
> > @v-jamcheung:
> > Your account lacks the required public GitHub organizations and permissions required of an internal contributor. Please review the Azure SDK onboarding documentation and use the associated Teams channel for support.
> > 
> > * [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
> > * [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_
> > 
> > You can verify the state of your account by running the [Validate-AzsdkCodeOwner](https://github.com/Azure/azure-sdk-tools/blob/main/tools/github/scripts/Validate-AzsdkCodeOwner.ps1) script from the Azure SDK tools repository.
> > Please also be sure to add yourself to [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS) for this library, if you will be maintaining it going forward.
> 
> Updated [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS).
> 
> As for the permissions, I believe I will not need write permissions as someone else will be assigned this PR and will merge it once I have addressed all the comments and gotten the necessary approvals.

You cannot appear in CODEOWNERS without the proper memberships and rights.   Whether you complete this PR or no, you'll need to request permissions.

**Comment by v-jamcheung** [comment]
@ArthurMa1978 @KrzysztofCwalina @tg-msft @archerzz @ArcturusZhang @pallavit @ronniegeraghty 

Hi team,

I am currently writing the tests for this .NET SDK.
Meanwhile please could people review the SDK code and leave comments if there are any issues.

Thanks.

**Comment by ronniegeraghty** [comment]
Checking a couple of things here. 
If @v-jamcheung is an internal contributor, I'll remove the `Community Contribution` label.

@v-jamcheung, can you tell me how the list of reviewers was added to this PR? 
Looking at our `CODEOWNERS` there is no specific listing for `/sdk/oracle/*` so it should have reverted to the owners for `/sdk/` which are @jsquire and @pallavit. Did you manually add the other reviewers listed?

**Comment by jsquire** [comment]
@ronniegeraghty:  It's a management library, which correctly routes to Arthur and team when there are no CODEOWNERS.

**Review by archerzz — COMMENTED** [review]
@v-jamcheung  could you please fix the errors in the CI pipeline? Thanks.

**Comment by v-jamcheung** [comment]
New PR containing regenerated SDK code after typespec PR (https://github.com/Azure/azure-rest-api-specs/pull/29305) was merged: https://github.com/Azure/azure-sdk-for-net/pull/44499

Putting playback testing on hold, will add them before releasing stable version.

**Pull request closed without merging**
